src/test/java/com/puppycrawl/tools/checkstyle/checks/whitespace/SeparatorWrapCheckTest.java

```diff
@@ -58,19 +58,19 @@ public void testDot()
         checkConfig.addAttribute("option", "NL");
         checkConfig.addAttribute("tokens", "DOT");
         final String[] expected = {
-            "31:10: " + getCheckMessage(MSG_LINE_NEW, "."),
+            "29:10: " + getCheckMessage(MSG_LINE_NEW, "."),
         };
-        verify(checkConfig, getPath("InputSeparatorWrap.java"), expected);
+        verify(checkConfig, getPath("InputSeparatorWrapForTestDot.java"), expected);
     }
 
     @Test
     public void testComma() throws Exception {
         checkConfig.addAttribute("option", "EOL");
         checkConfig.addAttribute("tokens", "COMMA");
         final String[] expected = {
-            "39:17: " + getCheckMessage(MSG_LINE_PREVIOUS, ","),
+            "35:17: " + getCheckMessage(MSG_LINE_PREVIOUS, ","),
```

Comment:
@sagar-shah94 
As was discussed above (https://github.com/checkstyle/checkstyle/pull/3915#discussion_r104325446)  it is better to to make changes to input files with small steps. So, for this issue just copy-pasting inout files will be enough. Sorry, that I forced you to make unnecessary changes.